R/use_cassette.R

```diff
@@ -175,17 +179,45 @@ use_cassette <- function(
     re_record_interval = re_record_interval,
     clean_outdated_http_interactions = clean_outdated_http_interactions
   )
-  if (is.null(cassette)) {
-    force(...)
-    return(NULL)
-  }
-  on.exit(cassette$eject())
-  cassette$call_block(...)
-  return(cassette)
+
+  # force all arguments
+  list(...)
+
+  invisible(cassette)
 }
 
-check_empty_cassette <- function(cas) {
```

Comment:
I deleted this function since it's not used anywhere. (The equivalent is now performed by `Casette$remove_empty_cassette()`)